docs-search/index/src/Docs/Search/IndexBuilder.js

```diff
@@ -1,13 +1,17 @@
 /* global __dirname require exports */
 
 import globMain from "glob";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
 
-export function getDirname () {
-  return __dirname;
-};
+export function getDirname() {
+  const fileName = fileURLToPath(import.meta.url);
+  const dirName = path.dirname(fileName);
+  return dirName;
+}
```

Comment:
`spago bundle` has a workaround for patching `__dirname` but it is only applied when bundling for platform=node.

I opted to just patch the source directly here though instead of hacking the `spago bundle` code to shim these for browsers.

Reply:
Yup this is good 👍